build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -0,0 +1,1520 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# TinyAra Configuration
+#
+
+#
+# Build Setup
+#
+# CONFIG_EXPERIMENTAL is not set
```

Comment:
Hi, it seems that some commits were merged while your PR was still open for review.

You need to fetch the latest code and re-apply your commits/changes on top of it and push again for consistency.